llama.cpp

```diff
@@ -15489,7 +15510,8 @@ static void llama_model_quantize_internal(const std::string & fname_inp, const s
             const int64_t nrows = tensor->ne[1];
 
             static const int64_t min_chunk_size = 32 * 512;
-            const int64_t chunk_size = n_per_row >= min_chunk_size ? n_per_row : n_per_row * ((min_chunk_size + n_per_row - 1)/n_per_row);
+            const int64_t chunk_size = (n_per_row >= min_chunk_size ? n_per_row : n_per_row * ((min_chunk_size + n_per_row - 1)/n_per_row)) *
+                                       chunk_size_multiplier;
```

Comment:
Not 100% sure if `chunk_size_multiplier` is really necessary. We can safely assume that `n_per_row` is always divisible by 8 so I think we can simplify and not add the `chunk_size_multiplier` logic at all. Unless I'm overlooking something

Reply:
We need to ensure that the number of rows (`nrows`) for each chunk is always divisible by 8 in order to consider 8 rows at a time for the interleaved weight layout. `n_per_row` only considers the elements of a row.